test/fixtures/3.0/ping/src/__tests__/acceptance/home-page.acceptance.ts

```diff
@@ -0,0 +1,31 @@
+import {Client} from '@loopback/testlab';
+import {PingApplication} from '../..';
+import {setupApplication} from './test-helper';
+
+describe('HomePage', () => {
```

Comment:
Can we move this test file to top-level `test` directory please? For example, `test/acceptance-loopback4.js`.